scripts/build/cardano-sl.sh

```diff
@@ -245,27 +254,40 @@ elif [[ $spec_prj == "sl" ]]; then
   to_build="cardano-sl"
 elif [[ $spec_prj == "lwallet" ]]; then
   to_build="cardano-sl-lwallet"
+elif [[ $spec_prj == "explorer" ]]; then
+  to_build="cardano-sl-explorer"
 elif [[ $spec_prj == "sl+" ]]; then
   to_build="cardano-sl cardano-sl-lwallet cardano-sl-tools"
 else
   to_build="cardano-sl-$spec_prj"
 fi
 
+# If the use didn't specify that he doesn't want explicitly to exclude Explorer, build it
```

Comment:
No, it shouldn't be here. If I pass only `sl` as argument, I want to build parts (a) and (b), but not (c). But with this logic also (c) will be built.